src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/SegmentedCpxVariantSimpleVariantExtractor.java

```diff
@@ -835,9 +835,10 @@ private void extractFrontAndRearInsertions(final VariantContext complexVC, final
             }
 
             if (firstRefSegmentIdx != altArrangement.size() - 1) {
-                final Allele anchorBaseRefAlleleRear = Allele.create(refBases[refBases.length - 2], true);
-                final SimpleInterval startAndStop = makeOneBpInterval(complexVC.getContig(), complexVC.getEnd());
-                final VariantContextBuilder rearIns = getInsFromOneEnd(false, firstRefSegmentIdx, startAndStop, anchorBaseRefAlleleRear, refSegmentLengths, altArrangement, true);
+                final int pos = complexVC.getEnd();
```

Comment:
Why is it the end of the VC here and not the start?

Reply:
The reason was because this is for extracting rear insertion.
Example:
```
SEGMENTS=chr1:100-200,chr1:200-300,chr1:300-400,chr1:400-500
ALT_ARRANGEMENT=chrOther:100-158,1,-3,4,UINS-433
```
Here we would emit four records:
a front insertion of 59 bases at position chr1:100, a deletion of chr1:200-300, an inversion of chr1:300-400, and a rear insertion of 433 bases at position of chr1:500.